api/v1alpha1/inclusterippool_types.go

```diff
@@ -41,6 +41,27 @@ type InClusterIPPoolSpec struct {
 
 // InClusterIPPoolStatus defines the observed state of InClusterIPPool.
 type InClusterIPPoolStatus struct {
+	// Addresses reports the count of total, free, and used IPs in the pool.
+	// +optional
+	Addresses InClusterIPPoolStatusIPAddresses `json:"ipAddresses,omitempty"`
```

Comment:
I meant turning this into a pointer, and removing optional and omitempty from the ints in the struct. Should've written that more clearly before, sorry.

Reply:
Ah, yes, ok. I've made the status a pointer and de-pointerized the counts. All seems seems good.